plugins/signature_provider_plugin/signature_provider_plugin.cpp

```diff
@@ -42,6 +42,9 @@ class signature_provider_plugin_impl {
       std::tuple<std::string, std::string, std::string> parse_spec(const std::string& spec) const {
          auto delim = spec.find("=");
          EOS_ASSERT(delim != std::string::npos, chain::plugin_config_exception, "Missing \"=\" in the key spec pair");
+         // public_key can be base64 encoded with trailing `=`
```

Comment:
Can you add in the comment what `spec` looks like when the public_key has trailing `=`?